lottie-ios.podspec

```diff
@@ -24,6 +24,7 @@ For the first time, designers can create and ship beautiful animations without a
 
   s.ios.deployment_target = '8.0'
   s.osx.deployment_target = '10.10'
+  s.tvos.deployment_target = '9.0'
 
   s.source_files = 'lottie-ios/Classes/**/*'
```

Comment:
It works without crashing even if I didn't performed any exhaustive tests to be 100% sure of that. :-)